Competition2017/src/competition/subsystems/agitator/commands/IntakeAgitatorCommand.java

```diff
@@ -0,0 +1,28 @@
+package competition.subsystems.agitator.commands;
+
+import org.apache.log4j.Logger;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.agitator.AgitatorSubsystem;
+import competition.subsystems.agitator.commands.IntakeAgitatorCommand;
+import xbot.common.command.BaseCommand;
+
+public class IntakeAgitatorCommand extends BaseAgitatorCommand {
```

Comment:
There's a direction for the agitator that sends balls towards the shooter and a direction that does the opposite, so I believe the logic here was intake is towards the shooter.